### DISCUSSION TITLE
Need base image for Win server 2016 (extended support)

### DISCUSSION THREAD

**Discussion by Alvin-Prasla** [dotnet/dotnet-docker#3250] [Q&A] [answered]
Which base image to use to have .NET 5.0 and VS build tools 2019 installed? 

More detail on Stackoverflow: [https://stackoverflow.com/questions/69649607/docker-image-for-windows-server-2016-with-framework-net-5-0](https://stackoverflow.com/questions/69649607/docker-image-for-windows-server-2016-with-framework-net-5-0)

Windows server 2016 support: [https://docs.microsoft.com/en-us/lifecycle/products/windows-server-2016](https://docs.microsoft.com/en-us/lifecycle/products/windows-server-2016)
Edition: Windows server 2016 Standard
Version: 1607
OS Build: 10.0.14393

**Comment by mthalman** [comment, marked as answer]
I've responded on Stack Overflow: https://stackoverflow.com/a/69739068/3174766

**Reply by Alvin-Prasla** [reply]
Thanks Matt for your help
